tests/src/test/java/alluxio/shell/command/TailCommandIntegrationTest.java

```diff
@@ -80,4 +80,12 @@ public void tailFileWithUserSpecifiedBytes() throws Exception {
     byte[] expect = BufferUtils.getIncreasingByteArray(48, 2000);
     Assert.assertArrayEquals(expect, mOutput.toByteArray());
   }
+
+  @Test
+  public void tailFileWithUserSpecifiedBytesWithUnit() throws Exception {
+    FileSystemTestUtils.createByteFile(mFileSystem, "/testFile", WriteType.MUST_CACHE, 10000);
+    mFsShell.run("tail", "-c", "2KB", "/testFile");
+    byte[] expect = BufferUtils.getIncreasingByteArray(0, 2048);
```

Comment:
will this work? `tail` command show the last 2048 byte here, 
I think it should be 
```java
byte[] expect = BufferUtils.getIncreasingByteArray(10000 - 2048, 2048);
```